index.html

```diff
@@ -325,9 +325,10 @@
       <dfn>register the observer</dfn> and must run these steps:</p>
       <ol data-link-for="PerformanceObserverInit">
         <li>Filter unsupported names within the `entryTypes` sequence, and
-        replace the `entryTypes` sequence with the new filtered sequence.</li>
-        <li>If the _options'_ <a>entryTypes</a> attribute is an empty sequence,
-        <a>throw</a> a JavaScript `TypeError`.
+        replace the `entryTypes` sequence with the new filtered sequence.
+        </li>
+        <li>If the resulting '<a>entryTypes</a>' sequence is an empty sequence, abort these steps.</li>
+        <p class='note'>User agents are encouraged to log this fact in a developer console, to aid debugging.</p>
```

Comment:
As a reference, looking at CSP, they actually make a normative "SHOULD" recommendation. 

> "The user agent SHOULD notify developers that a directive was ignored. A console warning might be appropriate, for example."

Perhaps we should do the same? Something like..

"... abort these steps. The user agent SHOULD notify developers that the observe call has aborted, due to empty '<a>entryTypes</a>' sequence. A console warning might be appropriate, for example."

Reply:
ignoring a CSP directive sounds more problematic noticing than failing to observe perf entries. it's probably worth asking around for opinions or not. I guess it depends on how strongly we feel. @yoavweiss , got an opinion as a dev?

Reply:
I think we should change the processing from filtering unsupported names to ignoring and logging them to console. A developer who is trying to observe "resource" and "navigation" and "longtaks" would want to know that "longtaks" entries are not coming anytime soon.

Reply:
agreed, but how prescriptive should we be in logging into the console? My PR suggests a non-norm note, @igrigorik suggests a normative SHOULD.

Reply:
My vote goes for a normative SHOULD.